tests/observatory/platform/utils/test_config_utils.py

```diff
@@ -71,83 +71,39 @@ def test_terraform_credentials_path(self):
     def test_find_schema(self):
         schemas_path = test_fixtures_path("schemas")
 
-        # Tests that don't use a prefix
-        table_name = "grid"
-
-        # 2020-09-21
-        release_date = pendulum.datetime(year=2015, month=9, day=21)
-        result = find_schema(schemas_path, table_name, release_date)
-        self.assertIsNone(result)
-
-        # 2020-09-22
-        expected_schema = "grid_2015-09-22.json"
-        release_date = pendulum.datetime(year=2015, month=9, day=22)
-        result = find_schema(schemas_path, table_name, release_date)
+        # Basic test case
+        expected_schema = "table_a.json"
+        result = find_schema(schemas_path, "table_a")
         self.assertIsNotNone(result)
         self.assertTrue(result.endswith(expected_schema))
 
-        # 2015-09-23
-        release_date = pendulum.datetime(year=2015, month=9, day=23)
-        result = find_schema(schemas_path, table_name, release_date)
+        # Expect to find dated table
+        expected_schema = "table_b_2000-01-01.json"
+        result = find_schema(schemas_path, "table_b")
         self.assertIsNotNone(result)
         self.assertTrue(result.endswith(expected_schema))
 
-        # 2020-04-28
-        expected_schema = "grid_2016-04-28.json"
-        release_date = pendulum.datetime(year=2016, month=4, day=28)
-        result = find_schema(schemas_path, table_name, release_date)
+        # Prefix test
+        expected_schema = "prefix_table.json"
+        result = find_schema(schemas_path, "table", prefix="prefix_")
```

Comment:
It would be good to add a test back that finds a table with a prefix, where there are two available dates, which is what the previous tests did.

The tests should check all of the combinations of: prefix, table_name and release_date.